internal/statestore/redis.go

```diff
@@ -319,6 +320,8 @@ func (rb *redisBackend) GetTicket(ctx context.Context, id string) (*pb.Ticket, e
 		return nil, status.Errorf(codes.Internal, "%v", err)
 	}
 
+	ticket.CreateTime = ptypes.TimestampNow()
```

Comment:
This should be in the frontend, not redis.
Additionally please add validation that create time is not set before setting it here, with appropriate testing (see https://github.com/googleforgames/open-match/pull/1160/files )

Reply:
Not sure what I was thinking there! :) Too late in the night may be  - Did not even add it to create :)! Thanks for pointing out. Fixed and added validation  (new case to the test you wrote)